scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -80,6 +79,7 @@ object TreeSyntax {
     }
   }
   private final class SyntaxInstances(dialect: Dialect) {
+    val keywords = scala.meta.internal.tokenizers.keywords(dialect)
```

Comment:
optional: since we already within `internal`, wouldn't `tokenizers.keywords(...)` work?

Reply:
It works! Changed